open_alchemy/__init__.py

```diff
@@ -8,6 +8,7 @@
 from sqlalchemy.ext import declarative
 
 from open_alchemy import types as oa_types
+from open_alchemy.build import PackageFormat
 
 from . import build as _build_module
```

Comment:
Is this a duplicate import? I like the leading `_`, the pattern elsewhere has been just `_build`. Do you think that should be changed?